drivers/rackspace/rackspace.go

```diff
@@ -62,6 +63,16 @@ func GetCreateFlags() []cli.Flag {
 			EnvVar: "OS_FLAVOR_ID",
 		},
 		cli.StringFlag{
+			Name:  "rackspace-net-id",
+			Usage: "Rackspace comma separated network ids the machine will be connected on",
+			Value: "",
+		},
+		cli.StringFlag{
```

Comment:
I think as long as you're consistent with your implementation, we should use the `StringSliceFlag` leaving `openstack-sec-groups` to get changed by someone else who needs it.